django_project/settings.py

```diff
@@ -47,8 +47,46 @@
 ]
 
 REST_FRAMEWORK = {
+    # Я бы вписал сюда CursorPagination, чтобы точно ну никто ене догадался что эе там в запросе,
+    # но отдельный класс я для этого создавать не хочу,
+    # так что будем считать PageNumberPagination тоже достаточно секьюрным
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
-    'PAGE_SIZE': 5,
+    'PAGE_SIZE': 6,
+}
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        },
+        'http_file': {
+            'class': 'logging.FileHandler',
+            'filename': os.path.join(BASE_DIR, 'logs/http_logs.log'),
+        },
+        'db_file': {
+            'class': 'logging.FileHandler',
+            'filename': os.path.join(BASE_DIR, 'logs/db_logs.log'),
+        },
+    },
+    'loggers': {
+        'django': {
+            'handlers': ['console'],
```

Comment:
UPD: Заметил, что все-таки логирования HTTP запросов не происходит. То что пишется в файл `http_logs` это работа родительского логгера, но там нет HTTP.  Логгер `django.request` логгирует только исключеня уровня ERROR и выше. Нужно чуть исправить
```suggestion
        'django.server': {
            'handlers': ['console', 'http_file'],
```